src/particles/text/component.tsx

```diff
@@ -64,26 +64,45 @@ export const getTextTag = (variant?: string): TextTag => {
 interface IStyledTextProps {
   theme: ITextTheme;
   alignment?: TextAlignment;
+  clipToLines?: number;
 }
 
 const StyledText = styled.span<IStyledTextProps>`
   ${(props: IStyledTextProps): string => themeToCss(props.theme)};
   ${(props: IStyledTextProps): string => (props.alignment ? `text-align: ${props.alignment}` : '')};
+
+  &.singleLine {
+    white-wrap: nowrap;
+    text-overflow: ellipses;
```

Comment:
These are for IE11 browsers, this will truncate the text to just a single line. And for browsers supporting `-webkit-line-clamp`, these properties would be ignored.

Reply:
pls add a comment for this, be clear which properties you are talkign about

Reply:
this was before I seperated the classes. So now, only singleLine class would work in IE11 and fixedLine is not supported.

Reply:
right - so please add a comment with this above the fixedLines class